StRoot/StPicoEvent/StPicoTrack.h

```diff
@@ -368,7 +368,14 @@ class StPicoTrack : public TObject {
   /// Parent vertex index. -2 if no vertex.
   Char_t   mVertexIndex;
 
-  ClassDef(StPicoTrack, 8)
+#if !defined (__TFG__VERSION__)
+  /// Fitted dN/dx
+  Float_t  mDnDx;
+  /// Fitted dN/dx error
+  Float_t  mDnDxError;
+#endif
+
+  ClassDef(StPicoTrack, 9)
```

Comment:
I told you this preprocessor branching would get you sooner or later... Not today though :-)

This new code does not make sense as in the same file you already have this:

https://github.com/star-bnl/star-sw/blob/9ee24f6838d1829043b2a3ffe0f81b034b02148c/StRoot/StPicoEvent/StPicoTrack.h#L321-L326

Reply:
Well, it does make sense. Otherwise the order of data memebers will change for the DEV version.

Reply:
You think the order is important? How?

Reply:
In general, it is not very important, but I try not to modify classes in the way data members are layouted in memory in the same way and not brake the order. The main pain in the neck may appear in the case when one has to add special rules for changes in the LinkDef file. Do you suggest to get rid of the preprocessor directive for dN/dx here?

Reply:
I still don't see what problem this is supposed to fix or prevent... I'd say remove it.